content/blog/eks-v3-release/index.md

```diff
@@ -0,0 +1,79 @@
+
+- [EKS Setup & Installation](https://www.pulumi.com/registry/packages/eks/installation-configuration/) - Instructions on installing and configuring the Pulumi EKS provider
+- [EKS How-to Guides](https://www.pulumi.com/registry/packages/kubernetes/how-to-guides/eks/) - Learn how to use the EKS provider to provision and manage Kubernetes clusters on AWS
+- [Pulumi AI](https://www.pulumi.com/ai) - Ask Pulumi AI to help you create a new EKS project
+
+## Key Highlights of EKS V3
+
+1. **Support for Amazon Linux 2023 (AL2023) and Bottlerocket Operating Systems**: Enhanced operating system options for node groups, allowing you to choose the OS that best fits your workloads and compliance needs. This addresses the upcoming deprecation of Amazon Linux 2 (AL2).
+
+2. **Access Entries for IAM Integration**: Enables replacement of the deprecated aws-auth ConfigMap with Access Entries for managing Kubernetes authentication.
+
+3. **EKS Managed Addons**: Simplified management of `vpc-cni`, `coredns`, and `kube-proxy` as EKS managed addons.
+
+4. **EKS Security Groups for Pods and Network Policies**: Enhanced network security and control within EKS clusters.
+
+## New Features and Improvements
+
+### Support for Amazon Linux 2023 and Bottlerocket
+
+We've expanded the operating system options for node groups in EKS v3 to address the upcoming deprecation of Amazon Linux 2 (AL2). You can now choose between Amazon Linux 2 (deprecated), Amazon Linux 2023 and Bottlerocket for your EKS nodes. This flexibility allows you to select the OS that best fits your workloads, security requirements, and compliance needs, while ensuring you're using a supported and actively maintained operating system. We've introduced a new `operatingSystem` property for node groups to facilitate this choice.
+
+### Access Entries for IAM Integration
+
+AWS has introduced Access Entries as a new method for granting IAM principals access to Kubernetes resources. This approach relies solely on AWS resources for managing Kubernetes auth, replacing the deprecated `aws-auth` ConfigMap. You can now leverage Access Entries by setting the `authenticationMode` to `API` in your cluster configuration.
+
+### EKS Managed Addons
+
+The EKS cluster components `vpc-cni`, `coredns`, and `kube-proxy` are now configured as EKS managed addons. This change simplifies management, especially for clusters with private API endpoints, and ensures that these critical components stay up to date automatically. Additionally it removes the dependency on `kubectl`, allowing pulumi-native management of clusters.
+
+### Cluster Autoscaler Integration
+
+Pulumi EKS v3 introduces better support for the Kubernetes Cluster Autoscaler. A new `ignoreScalingChanges` parameter for node groups allows Pulumi to ignore external scaling changes, facilitating seamless integration with dynamic scaling solutions.
+
+### EKS Security Groups for Pods and Network Policies
```

Comment:
Would be pretty cool to link to minimal examples if possible I think.

Reply:
Yeah good idea, I started creating some for those new features!